src/app/app.spec.e2e.ts

```diff
@@ -90,6 +90,40 @@ context(`EJawa demo`, () => {
         });
       });
 
+      it(`should be able to go from a spaceship to a speeder AND back and restore the original value`, () => {
+        DOM.list.elements.cy.eq(0).click();
+        DOM.list.elements.cy.eq(1).click();
+        DOM.list.elements.cy.eq(0).click();
+
+        const x = hardCodedListings[0] as VehicleListing;
```

Comment:
[minor]

```suggestion
        const vehicle = hardCodedListings[0] as VehicleListing;
```

Reply:
uh sure, I copied the above spec nearly exactly so I should refactor the whole file really to fix that correctly.